### PR TITLE
update shape of term orderby

### PR DESCRIPTION
**Pull request by hsimah**
- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
--------------------------------------------------
Fixes the disparity between input args for ordering terms and posts.

Does this close any currently open issues?
------------------------------------------
#328 

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
n/a

Any other comments?
-------------------
n/a

Where has this been tested?
---------------------------
**Operating System:** Windows, RHEL

**WordPress Version:** 5 (latest minor)

### CODE REVIEW

**Comment by hsimah** [comment]
> ERROR: Encountered errors while bringing up the project.

Not sure how to fix this.

**Comment by jonlow** [comment]
+1

File: src/TypeRegistry.php [hunk 1]
```diff
@@ -81,51 +81,52 @@ public static function init() {
...
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Taxonomy.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/TaxonomyEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/User.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/UsersConnectionSearchColumnEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/UserRole.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Enum/UserRoleEnum.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/SettingGroup.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/PostObject.php';
-		require_once WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/TermObject.php';
+		require_once( WPGRAPHQL_PLUGIN_DIR . 'src/Type/Object/Avatar.php' );
```
Comment (jonlow) [inline src/TypeRegistry.php]: Any reason for adding the brackets here? Seems like this is what's causing CI to fail

Comment (hsimah) [reply src/TypeRegistry.php]: not sure where they came from - I've recently started working on a macbook and I assume my IDE thought they were needed for PHP.

**Comment by jasonbahl** [comment]
For some strange reason Travis is not liking this PR.

I'm pulling it locally to check it out

**Comment by jasonbahl** [comment]
Ok, so I pulled this locally to do some tests, and it looks like this is breaking some things. 

For example, I ran the following query: 

```
{
  tags(first: 1, after: null, where: {orderby: [{field: COUNT, order: ASC}]}) {
    pageInfo {
      endCursor
      hasNextPage
    }
    nodes {
      name
      count
    }
  }
}
```

And I received the following response (expected):

```
{
  "data": {
    "tags": {
      "pageInfo": {
        "endCursor": "YXJyYXljb25uZWN0aW9uOjM=",
        "hasNextPage": true
      },
      "nodes": [
        {
          "name": "Another Test",
          "count": null
        }
      ]
    }
  }
}
```

But then, I used the `endCursor` as the input for `after` and ran the query again like so: 

```
{
  tags(first: 1, after: "YXJyYXljb25uZWN0aW9uOjM=", where: {orderby: [{field: COUNT, order: ASC}]}) {
    pageInfo {
      endCursor
      hasNextPage
    }
    nodes {
      name
      count
    }
  }
}
```

This causes a fatal error. Behind the scenes I'm seeing: 

```
WordPress database error:
[Unknown column &#039;t.count&#039; in &#039;where clause&#039;]

SELECT  t.term_id, tt.parent, tt.count, tt.taxonomy FROM wp_terms AS t  INNER JOIN wp_term_taxonomy AS tt ON t.term_id = tt.term_id WHERE tt.taxonomy IN (post_tag) AND tt.parent = 0; AND t.count > 0 ORDER BY tt.count ASC
```

I believe this might be related to needing a CursorBuilder implemented for term queries as it has been implemented for users and posts. 🤔

**Comment by hsimah** [comment]
I have family visiting for the next two weeks so will not be looking at this. Happy for someone else to claim and update, otherwise will get stuck into it asap :)

**Comment by jasonbahl** [comment]
@hsimah thanks for the update! I'll see if I can get it to the finish line while you have family in town.

**Pull request closed without merging**